app/views/components/summaryCardLPP.scala.html

```diff
@@ -43,9 +43,9 @@
 }
 
 @calculationContent = {
-    <a class="govuk-link" href="@{
-        controllers.routes.CalculationController.onPageLoad(summaryCard.principalChargeReference, summaryCard.penaltyCategory.toString).url
-    }">
+    <a class="govuk-link" aria-label="@{messages("summaryCard.appealLink.aria.label.lpp.vatPaid", messages(s"summaryCard.appealLink.aria.label.lpp.vatPaid.${summaryCard.penaltyCategory}"), summaryCard.dueDate)}" href="@{
```

Comment:
The message we're using for the label should be "View calculation of <first|second> late payment penalty for charge due on <date>" rather than the message you've called here. Also, could you combine this with the @ariaLabelText code on line 33?

Reply:
I've made some changes but i don't think i can combine with the @ariaLabelText code